packages/nixos/system.nix

```diff
@@ -105,10 +105,13 @@
   # string "perl" or "python".
   system.forbiddenDependenciesRegexes = [
     "perl"
-  ]
-  ++ lib.optionals (!config.contrast.debug.enable) [
-    "python" # Some of the debug packages need Python.
   ];
+  # TODO(katexochen): python dependency was added to iproute2 in https://github.com/NixOS/nixpkgs/pull/424834
+  # and removed again in https://github.com/NixOS/nixpkgs/pull/445799.
+  # Waiting for it to land in unstable.
+  # ++ lib.optionals (!config.contrast.debug.enable) [
+  #   "python" # Some of the debug packages need Python.
+  # ];
```

Comment:
Doesn't this do the exact opposite of what's described? 😅 

when `debug` is `true`:
```
optionals false [ "python" ] => []
```

when `debug` is `false`:
```
optionals true [ "python" ] => [ "python" ]
```

And if so, do we need python at all here?